deploy/manifests/sti-dev-v2/us-east-2/cluster/promtail/kustomization.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+namespace: promtail
```

Comment:
I was hoping log forwarding mechanism in general will live in the centralised GitOps-root repo since I can't imagine a K8S cluster that does not want it.

Reply:
Yes, this will be moved out as a part of storetheindex v3